iceoryx_binding_c/include/iceoryx_binding_c/chunk.h

```diff
@@ -39,6 +39,26 @@ void* iox_chunk_header_to_user_header(iox_chunk_header_t* const chunkHeader);
 /// @return const pointer to the user-header
 const void* iox_chunk_header_to_user_header_const(const iox_chunk_header_t* const chunkHeader);
 
+/// @brief The size of the whole chunk, including the header
+/// @return the chunk size
+uint32_t iox_chunk_header_user_chunk_size(const iox_chunk_header_t* const chunkHeader);
```

Comment:
s/user_//

Reply:
Yes, please remove the `user_`. The terminology is `chunk-header` and `chunk-payload` and the `chunk-payload` is further segmented into `user-header` and `user-payload`. According the documentation and the the implementation this returns the full chunk size.

What do you think about adding a `get`, e.g. `iox_chunk_header_get_chunk_size`?

@elfenpiff since you created most of the C API what do you think about the naming?

Reply:
I am a friend of the `get` and would prefer it over `iox_chunk_header_chunk_size` ... this sounds a  little bit weird.